src/single_byte.rs

```diff
@@ -645,6 +645,7 @@ mod tests {
     fn test_single_byte_decode() {
         decode_single_byte(IBM866, &data::SINGLE_BYTE_DATA.ibm866);
         decode_single_byte(ISO_8859_10, &data::SINGLE_BYTE_DATA.iso_8859_10);
+        if cfg!(miri) { return; } // Miri is too slow
```

Comment:
This (and the other change in this file) are editing code that a comment says is auto-generated and should not be edited. I'll need to figure out how to adjust the auto-generation accordingly.

Reply:
The loop that generates this is at https://github.com/hsivonen/encoding_rs/blob/adcb9b300428e9a06a5074c1cb1f23550c93ee01/generate-encoding-data.py#L1449

Reply:
Thanks! I adjusted the code generation, though there might be prettier way to do this.

This causes diffs in some other fails, looks like it runs rustfmt on the entire repository. But there's no diff in the single_byte tests.